library/init/coe.lean

```diff
@@ -4,7 +4,12 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura
 -/
 
-/-
+prelude
+
+import init.data.list.basic init.data.subtype.basic init.data.prod
+
+/-! # Coercions and lifts
+
 The elaborator tries to insert coercions automatically.
 Only instances of has_coe type class are considered in the process.
```

Comment:
At risk of creeping the scope here:
```suggestion
Only instances of `has_coe` type class are considered in the process.
```
And similarly for the other names in this comment.